tools/ci/success-http-headers.txt

```diff
@@ -1,7 +1,7 @@
   Self-signed certificate encountered.
     WARNING: certificate common name ‘127.0.0.1’ doesn't match requested host name ‘localhost’.
   HTTP/1.1 302 Found
-  Server: nginx/1.14.0 (Ubuntu)
+  Server: nginx_version
```

Comment:
Let's write this as `{nginx_version_string}`, so that it's clear it's a variable.  And maybe rename this to `success-http-headers.template.txt`.